src/composables/useWindowingConfig.ts

```diff
@@ -1,30 +1,43 @@
-import useWindowingStore from '@/src/store/view-configs/windowing';
+import { useWindowingStore } from '@/src/store/view-configs/windowing';
 import { Maybe } from '@/src/types';
 import type { Vector2 } from '@kitware/vtk.js/types';
 import { MaybeRef, unref, computed } from 'vue';
+import { useImageStatsStore } from '@/src/store/image-stats';
 
 export function useWindowingConfig(
   viewID: MaybeRef<string>,
   imageID: MaybeRef<Maybe<string>>
 ) {
   const store = useWindowingStore();
-  const config = computed(() => store.getConfig(unref(viewID), unref(imageID)));
+  const imageStatsStore = useImageStatsStore();
+  const config = computed(() => {
+    const imageIdVal = unref(imageID);
+    if (!imageIdVal) return undefined;
+    const viewIdVal = unref(viewID);
+    if (!viewIdVal) return undefined;
+    return store.getConfig(viewIdVal, imageIdVal).value;
+  });
 
   const generateComputed = (prop: 'width' | 'level') => {
     return computed({
-      get: () => config.value?.[prop] ?? 0,
+      get: () => {
+        return config.value?.[prop] ?? 0;
+      },
       set: (val) => {
         const imageIdVal = unref(imageID);
         if (!imageIdVal || val == null) return;
-        store.updateConfig(unref(viewID), imageIdVal, { [prop]: val });
+        store.updateConfig(unref(viewID), imageIdVal, { [prop]: val }, true);
```

Comment:
Not sure what this `true` parameter is doing here. Might this be accidentally added?

Reply:
Thats the "user did this", don't reset when fully loaded flag.

Reply:
Ah, I missed that somehow. Thanks!